src/deluge/gui/views/view.cpp

```diff
@@ -874,10 +874,8 @@ void View::modEncoderAction(int32_t whichModEncoder, int32_t offset) {
 				//if current knobPos exceeds 127, e.g. it's 128, then it needs to drop to 126 before a value change gets recorded
 				//if newKnobPos exceeds 127, then it means current knobPos was 127 and it was increased to 128. In which case, ignore value change
 				if ((getRootUI() == &instrumentClipView) || (getRootUI() == &automationInstrumentClipView)) {
-					if (clip->output->type == InstrumentType::MIDI_OUT) {
-						if ((knobPos == 64) || (newKnobPos == 64)) {
-							return;
-						}
+					if ((clip->output->type == InstrumentType::MIDI_OUT) && (newKnobPos == 64)) {
```

Comment:
clamp the value instead, this will cause bugs if the offset is greater than 1

std::clamp(newKnobPos, -64, 63)